doc/design/upgrade.md

```diff
@@ -38,14 +59,16 @@ If anything wrong happens during upgrade, vic-machine will rollback VCH to origi
 
 ### Downgrade
 
-For any running VCH, VIC admin can downgrade existing VCH to an old version. (I assume this function is only used for upgrade roll back, we might not want to recommend downgrade for anything working well, then the downgrade scope will be same to upgrade versions)
-
-Verified docker-engine upgrade/downgrade, which works well between 1.11.2 and 1.12. The running container will be stopped after upgrade/downgrade, but is good to start again in new version. And all images and containers information is not lost. So from technical point of view, we should support same story.
+Verified docker-engine upgrade/downgrade, which works well between 1.11.2 and 1.12. The running container will be stopped after upgrade/downgrade, but is good to start again in new version. And all images and containers information is not lost. But think of the complexity to support version downgrade, we'll not go with this at this moment, unless there is explicity user requirement.
```

Comment:
I see your point. Please skip my reply above.
For this, I think we should limit the roll back function. If there is new version container created already, we should fail the rollback, and let user delete new container before rollback. As after rollback, most likely the new container will not function well, user should make decision on that.